docs/_sass/callouts.scss

```diff
@@ -0,0 +1,63 @@
+@import 'colors';
+
+.callout {
+  margin-top: 5px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding: 10px 20px;
+  color: $glue-grey-800;
+}
+
+.important {
+  @extend .callout;
+  background-color: $glue-yellow-50;
+  border: 1px solid $glue-yellow-200;
+}
+
+.note {
+  @extend .callout;
+  background-color: $glue-blue-50;
+  border: 1px solid $glue-blue-200;
+}
+
+.tldr {
```

Comment:
which of these do we actually use? I don't see .tldr referenced anywhere for example

Reply:
We don't use them yet, but I thought they could be useful to incorporate.

Reply:
I will remove the callout stuff since it's not used.